torq.q

```diff
@@ -611,16 +611,29 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+// function to execute functions in .proc.initlist
 .proc.try:{[id;a]
-  .lg.o[id;"attemping to run: ",.Q.s1 a];
-  @[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a]}[id;a]];
+  .lg.o[id;"attempting to run: ",.Q.s1 a];
+  success:@[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a];:1b}[id;a]];
+  if[-1h=type success;:()];
+  .proc.initexecuted,:a;
+  .lg.o[id;"run successful: ",.Q.s1 a];
+ }
+
+.proc.initcmd:{
+  // command line functions executed first
+  $[`initlist in key .proc.params;
+    {.proc.try[`init;(x;`)]}each .proc.params`initlist;
+    .lg.o[`init;"no initialisation functions found in cmd line args"]
+   ];
  }
 
-// function to execute functions in .proc.initlist
 .proc.init:{
-  if[0=count .proc.initlist;:.lg.o[`init;"no initialisation functions found"]];
-  .proc.try[`init]each .proc.initlist;
-  .proc.initexecuted,:.proc.initlist;
+  .proc.initcmd[];
+  $[count .proc.initlist;
```

Comment:
Looks like you've only added line 632 so I think you should revert the rest of the changes to this function.